### ISSUE TITLE
More detailed debriefing

### ISSUE THREAD

**Issue by shomke** [dcs-liberation/dcs_liberation#875]
More detailed debrief report would be very interesting.
Could be in a form of a simple table with the following columns.

Package/mission - Flight name - Airbase - Aircraft type - Nb of aircraft - Nb of aircraft lost

**Comment by HILOK** [comment]
yes, and a little MISSION SUCCESS / PARTIAL SUCCESS / UNSUCCESSFUL message as in the early days would be nice : )
